raizama/core/tests/test_command_import_spreadsheet.py

```diff
@@ -0,0 +1,219 @@
+        )
+        self.non_csv_file = "WrongFilePath.txt"
+
+    def test_existance_custom_command(self):
+        self.assertIsNone(call_command("importspreadsheet", self.csv_path))
+
+    def test_exception_non_csv_file(self):
+        self.assertRaises(
+            FileNotFoundError, call_command, "importspreadsheet", self.non_csv_file
+        )
+
+    def test_number_of_organizations(self):
+        call_command("importspreadsheet", self.csv_path)
+        expected_number_orgs = 6
+        output_orgs = Organization.objects.count()
+        self.assertEqual(
+            output_orgs,
+            expected_number_orgs,
+            msg=f"Expected {expected_number_orgs}, but got this {output_orgs}",
+        )
+
+    def test_name_of_first_organization(self):
+        call_command("importspreadsheet", self.csv_path)
+        expected_name = "Uma Doce Revolução"
+        output_name = Organization.objects.first().name
+        self.assertEqual(
+            output_name,
+            expected_name,
+            msg=f"Expected {expected_name}, but got this {output_name}",
+        )
+
+    def test_name_of_last_organization(self):
+        call_command("importspreadsheet", self.csv_path)
+        expected_city = "Copirecê"
+        outpu_city = Organization.objects.last().name
+        self.assertEqual(
+            outpu_city,
+            expected_city,
+            msg=f"Expected {expected_city}, but got this {outpu_city}",
+        )
+
+    def test_city_of_last_oragnization(self):
+        call_command("importspreadsheet", self.csv_path)
+        expected_city = "Irecê"
+        output_city = Organization.objects.last().city
+        self.assertEqual(
+            output_city,
+            expected_city,
+            msg=f"Expected {expected_city}, but got this {output_city}",
+        )
+
+    def test_filter_duplicate(self):
+        call_command("importspreadsheet", self.csv_path)
+        out = StringIO()
+        call_command("importspreadsheet", self.csv_path, stdout=out)
+        self.assertIn("Added 0 new organization(s)", out.getvalue())
+
+    def test_address_of_all_organizations(self):
+        call_command("importspreadsheet", self.csv_path)
+        dict_address = {
+            "Uma Doce Revolução": "-",
+            "Troço Bom": "-",
+            "Super Vegan": "-",
+            "Refeitório Orgânico":
+                "Rua Dezenove de Fevereiro, 120,"
+                " Botafogo, Rio de Janeiro, BR 22280-030",
+            "Projeto Ecovila Muriqui Assú":
+                "Estr. Aristide Melo, 662,"
+                " Pendotiba, Niterói - Rio de Janeiro (RJ)",
+            "Copirecê": "Rua São Francisco, 165 - Copirecê - Irecê - Bahia - 44.900-000",
+        }
+        for org, address in dict_address.items():
+            queryset_result = Organization.objects.filter(name__iexact=org).values(
+                "street"
+            )
+            output_address = queryset_result[0].get("street")
```

Comment:
Ao invés de fazer a query toda para pegar o primeiro, faça isso direto na query:

```suggestion
            output_address = queryset_result.first().get("street")
```


Pode mudar isso nos testes seguitnes tb?